recipes/scikit-bio/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: scikit-bio
+  version: {{ version }}
+
+source:
+  fn: scikit-bio-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
+  md5: bbbe704410c59083e9eb65fa9d01a50d
+
+build:
+    number: 0
```

Comment:
Please add the following to `build`.

```
  skip: true  # [win]
```

Also please feel free to include `[skip appveyor]` in subsequent commits to stop that CI from building.